### PR TITLE
Add blog post

### PR DESCRIPTION
**Pull request by amoralej**
Feedback is appreciated.

### CODE REVIEW

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+tags:
+date: 2017-03-22 19:22:55 CET
+---
+
+
+
+When new contributors join RDO, they ask for recommendations about
+how to add new services and help RDO users to adopt it. This post is
+not a official policy document nor a detailed description about how to carry
+out some activities but provides some high level recommendations to newcomers
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: Missing comma before "but provides"

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+based on what I have learned and observed in the last year working in RDO.
+
+Note that you are not required to follow all these steps and even you can
+have your own ideas about it. If you want to discuss it, let us know your thoughts, we are always open to improvements.
+
+### 1. Adding the package to RDO
+
+The first step is to add the package(s) to RDO repositories as shown
+in [RDO documentation](https://www.rdoproject.org/documentation/rdo-packaging/#how-to-add-a-new-package-to-rdo-trunk).
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: You can also reference https://www.rdoproject.org/what/new-package/

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+This tipically includes the main service package, client library and maybe
+a package with a plugin for horizon. In some cases you may need to add new
+libraries to Fedora which are required for the new packages.
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: I'd add some detail here, explaining why we add non-OpenStack, common dependencies to Fedora first.

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+### 2. Create a puppet module
+
+Although there are multiple deployment tools for OpenStack based on several
+frameworks, the use of puppet is widely used by different tools or even directly
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: "the use of" is redundant, I'd remove it.

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+As explained in [a previous post](https://www.rdoproject.org/blog/2017/03/rdo-ci-in-a-nutshell/)
+we run several jobs in RDO CI to validate the content of our repos. Most
+of the times the first way to get it tested is by adding the new service
+to one of the puppet-openstack-integration scenarios which is also
+recommended to get the puppet module tested in upstream gates. An example
+of how to add a new service into p-o-i is in [this review](https://review.openstack.org/#/c/429705/).
+
+### 4. Adding deployment support in Packstack
+
+If you want to make easier for RDO users to evaluate a new service, adding
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: make it easier

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+### Other recomendations
+
+#### Follow OpenStack governance policies
+
+RDO methodology and tooling is conceived according to OpenStack upstream
+release model, so following policies about [release management](https://github.com/openstack/releases/blob/master/README.rst)
+and [requirements](https://github.com/openstack/requirements/blob/master/README.rst)
+will help to make package maintenance easier in RDO.
+
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: Is it here where we ask projects to do proper tagging? We could do it a bit more explicitly :)

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+#### Advertise your work to the RDO community
+
+Making potential users aware of availability of new services or other
+improvements is a good practice. RDO provides several ways to do this as
+sending mails to [our mailing lists](https://www.rdoproject.org/community/mailing-lists/),
+writing a post in [the blog](https://www.rdoproject.org/blog/), adding
+references in our documentation, creating screencast demos, etc...
```
Comment (javierpena) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: +joining our weekly IRC meetings

File: source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md (hunk 6 above)
Comment (amoralej) [inline source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md]: Yes, that was the idea  :)

**Comment by javierpena** [comment]
It looks good to me now.

**Pull request merged**